src/libs/actions/IOU.js

```diff
@@ -0,0 +1,14 @@
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../ONYXKEYS';
+
+/**
+ * Retrieve the users preferred currency
+ */
+export default function getPreferredCurrency() {
+    Onyx.merge(ONYXKEYS.IOU, {isLoadingCurrency: true});
+
+    // fake loading timer, to be replaced with actual network request
+    setTimeout(() => {
+        Onyx.merge(ONYXKEYS.IOU, {isLoadingCurrency: false});
+    }, 1600);
```

Comment:
Merging this for two reasons: 
- Provides an example for external contributors to replace/build on top of
- Allows us to verify additional tests regarding state

Reply:
This should definitely be `merge()` or else it blows away all other properties of the `IOU` key.